src/libmongoc/doc/ref/packages.rst

```diff
@@ -0,0 +1,200 @@
+
+.. _EPEL: https://docs.fedoraproject.org/en-US/epel/
+.. _conan: https://conan.io/
+.. _vcpkg: https://vcpkg.io/
+
+|libbson| and |libmongoc| are available from several package management tools
+on a variety of systems.
+
+.. important::
+
+  The third-party packages detailed here are not directly controlled via the
+  |mongo-c-driver| maintainers, and the information found here may be incomplete
+  or out-of-date.
+
+
+Package Names and Availability
+******************************
+
+This table details the names and usage notes of such packages.
+
+.. seealso::
+
+  For a step-by-step tutorial on installing packages, refer to
+  :doc:`/learn/get/installing`.
+
+
+.. list-table::
+  :name: Available Packages
+  :header-rows: 1
+  :align: left
+  :widths: auto
+
+  - - Packaging Tool
+    - Platform(s)
+    - |libbson| package(s)
+    - |libmongoc| package(s)
+    - Notes
+
+  - - :index:`APT <!pair: APT; package names>` (``apt``/``apt-get``)
+    - Debian-based Linux distributions (:index:`Debian`, :index:`Ubuntu`, Linux
+      Mint, etc.)
+    - ``libbson-1.0-0``, ``libbson-dev``,
+      ``libbson-doc``
+    - ``libmongoc-1.0-0``, ``libmongoc-dev``,
+      ``libmongoc-doc``
+    - The ``libmongoclient`` packages are for the C++ database driver and will
+      transitively depend on ``libmongoc``.
+
+  - - :index:`Yum <!pair: Yum; package names>`/:index:`DNF <!pair: DNF; package names>`
+    - RHEL-based systems (RHEL, :index:`Fedora`, :index:`CentOS`,
+      :index:`Rocky Linux`, :index:`AlmaLinux`)
+    - ``libbson``, ``libbson-devel``
+    - ``mongo-c-driver-libs``, ``mongo-c-driver-devel``
+    - *Except on Fedora* the EPEL_ repositories must be enabled (i.e. install
+      the ``epel-release`` package first)
+
+  - - :index:`APK <!pair: APK; package names>`
+    - :index:`Alpine Linux <!pair: Alpine Linux; package names>`
+    - ``libbson``, ``libbson-dev``, ``libbson-static``
+    - ``libmongoc``, ``libmongoc-dev``, ``libmongoc-static``
+    - .. empty cell
+
+  - - :index:`pacman <pair: !package names; pacman>`
+    - :index:`Arch Linux <pair: !package names; Arch Linux>`
+    - ``mongo-c-driver``
+    - ``mongo-c-driver``
+    - A single package provides both runtime and development support for both
+      |libbson| and |libmongoc|
+
+  - - :index:`Homebrew <!pair: Homebrew; package names>`
+    - :index:`macOS <!pair: macOS; package names>`
+    - |ditto|
+    - |ditto|
+    - |ditto|
+
+  - - :index:`Conan <!pair: Conan; package names>`
+    - Cross-platform
+    - |ditto|
+    - |ditto|
+    - |ditto|
+
+      See: :ref:`ref.conan.settings`
+
+  - - :index:`vcpkg <!pair: vcpkg; package names>`
+    - Cross-platform
+    - ``libbson``
+    - ``mongo-c-driver``
+    - See: :ref:`ref.vcpkg.features`
+
+
+.. - This prevents the quote-pair rewriting if we use had written two U+0027's directly:
+.. |ditto| unicode:: U+0027 U+0027
+
+.. index:: !Conan; Settings and Features
+.. _ref.conan.settings:
+
+Conan Settings and Features
```

Comment:
As far as I can find, those parametres are not described or documented outside of the source itself. Since they correspond to our own build settings, I doubt they will be changing out of lock-step with out own releases. With all that considered, and that we are going to support these tools as installation routes, I decided to document them here. Thoughts?

Either way, the links to the recipes will be useful to include.

Reply:
Considering the options are unlikely to change, documenting here seems OK, and may help readers. Resolving.